Source/WebCore/animation/AnimationTimeline.h

```diff
@@ -36,6 +36,24 @@ namespace WebCore {
 
 class WebAnimation;
 
+struct TimelineScope {
+    bool operator==(const TimelineScope& other) const
+    {
+        if (type == Type::Ident)
+            return other.type == Type::Ident && string == other.string;
+        return type == other.type;
+    }
+
+    enum class Type : uint8_t {
+        None,
+        All,
+        Ident
+    };
```

Comment:
You can put all of those on a single line.

Reply:
Fixed.